### ISSUE TITLE
IonQuant configuration fails in v18.0

### ISSUE THREAD

**Issue by bapalanski** [Nesvilab/FragPipe#699]
I'm trying to setup the new FragPipe v18 but keep running into an issue with the IonQuant configuration. After I click "download/update ", I get a java.net.SocketTimeoutException: Read timed out (see screenshot below)

If I try to manually navigate to the IonQuant.jar file, it tells me that it is not an IonQuant file (see screenshot below)

The new version looks great! Looking forward to trying it.
<img width="638" alt="Screen Shot 2022-06-01 at 8 06 51 AM" src="https://user-images.githubusercontent.com/105822212/171401464-b0a22043-af3b-4d6c-870d-6fb8113c90fb.png">
<img width="1020" alt="Screen Shot 2022-06-01 at 8 11 39 AM" src="https://user-images.githubusercontent.com/105822212/171401499-77109b2d-b8da-4ca1-b9b8-90b42c029d3c.png">

**Comment by fcyu** [comment]
Your network is not good so that FragPipe could not finish the downloading. Can you try again or troubleshoot your network?

Do you have any issue downloading MSFragger?

Best,

Fengchao

**Comment by fcyu** [comment]
Yes, you can download a standalone version from https://msfragger.arsci.com/ionquant/ . It is bigger than what FragPipe needs because there are additional libraries bundled. Everything else is the same.

Best,

Fengchao

**Comment by bapalanski** [comment]
Thank you! When I downloaded from here, FragPipe recognized the .jar file without any issue.
